docs/tutorials/enterprise_beginner/5_create_newsletter_form.md

```diff
@@ -0,0 +1,220 @@
+|Captcha|CAPTCHA|—|—|
+|Button|Sign up!|—|Action: Show a message</br>Message to display: Thank you!|
+
+The configuration should look like this:
+
+![Adding Fields to Newsletter Form](img/enterprise_tut_form_creation.png "Adding Fields to Newsletter Form")
+
+When you add all the fields, go to the previous menu and click **Publish**.
+Now you can edit the Front Page and add a Form block below the Random block.
+Edit the block and select the form you created. Click **Submit**.
+
+The Page should refresh with the Form block.
+
+![Newsletter Form Block](img/enterprise_tut_first_form.png "Raw Newsletter Form Block")
+
+It clearly differs from the page design, so you also need to customize the block's layout.
+
+### Change the block template
+
+First, add a new template for the Form block to align it with the Random block design.
+Create an `templates/blocks/form/newsletter.html.twig` file:
+
+``` html+twig hl_lines="1"
+<div class="row">
+    <div class="block-form {{ block_class }}">
+        {{ fos_httpcache_tag('relation-location-' ~ locationId) }}
+        {{ render(controller('ez_content:viewLocation', {
+            'locationId': locationId,
+            'viewType': 'embed'
+        })) }}
+        <style type="text/css">{{ block_style|raw }}</style>
+    </div>
+</div>
+```
+
+This template extends the default block layout by adding an additional class (line 1) that shares CSS styling with the Random block.
+
+Append the new template to the block by adding it to `config/packages/ezplatform_page_fieldtype.yaml`.
+Add the following configuration under the `blocks` key at the same level as other block names, e.g. `random`:
+
+``` yaml hl_lines="3"
+blocks:
+#...
+    form:
+        views:
+            newsletter:
+                template: blocks/form/newsletter.html.twig
+                name: Newsletter Form View
+```
+
+Now you have to apply the template to the block.
+Go back to editing the Page.
+Edit the Form block again. 
+In the **Design** tab, select the **Newsletter Form View** and click **Submit**.
+
+The block remains unchanged, but the results will be visible when you add CSS styling.
+
+### Change the field template 
+
+At this point, you need to change the field template.
+This results in alternating the position and design of the Form fields.
+
+Create an `templates/form_field.html.twig` file:
+
+``` html+twig
+{% block ezform_field %}
+    {% set formValue = field.value.getForm() %}
+    {% if formValue %}
+        {% set form = formValue.createView() %}
+        {% form_theme form 'bootstrap_4_layout.html.twig' %}
+        {% spaceless %}
+            {% if not ez_field_is_empty(content, field) %}
+                {{ form(form) }}
+            {% endif %}
+        {% endspaceless %}
+    {% endif %}
+{% endblock %}
+```
+
+Next, assign the template to the Page.
+In `config/packages/views.yaml`, at the same level as `pagelayout` and `user`, add:
+
+``` yaml
+ezpublish:
+    system:
+        site_group:
```

Comment:
The config in views.yaml is now `ezpublish.system.site`, not `site_group`, per previous steps.
Also, I think it's misleading when you repeat `ezpublish.system.<siteaccess>` but then tenn the reader to add it *under* `<siteaccess>`